qiskit/transpiler/passes/optimization/consolidate_blocks.py

```diff
@@ -49,12 +52,16 @@ def __init__(
     ):
         """ConsolidateBlocks initializer.
 
+        The decomposer used is kak_basis_gate if it has non-None value. Otherwise
+        the decomposer is basis_gates if it has non-None value. If both are None,
+        then a default decomposer is used.
+
         Args:
```

Comment:
There doesn't appear to be a decomposer any more.

Reply:
A decomposer is still used. But this docstring needs improvement.

Reply:
Docstring improved in bf0a1f4. You probably meant that the decomposer is no longer part of the API. I removed mention of it.